delta/plugins/blazegraph/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/slowqueries/BlazegraphSlowQueryStoreSuite.scala

```diff
@@ -0,0 +1,77 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.blazegraph.slowqueries
+
+import ch.epfl.bluebrain.nexus.delta.plugins.blazegraph.slowqueries.model.BlazegraphSlowQuery
+import ch.epfl.bluebrain.nexus.delta.rdf.IriOrBNode.Iri
+import ch.epfl.bluebrain.nexus.delta.rdf.query.SparqlQuery
+import ch.epfl.bluebrain.nexus.delta.sdk.views.ViewRef
+import ch.epfl.bluebrain.nexus.delta.sourcing.model.{Identity, Label, ProjectRef}
+import ch.epfl.bluebrain.nexus.testkit.IOFixedClock
+import ch.epfl.bluebrain.nexus.testkit.bio.BioSuite
+import ch.epfl.bluebrain.nexus.testkit.postgres.Doobie
+import munit.AnyFixture
+
+import java.time.{Duration, Instant}
+import scala.concurrent.duration.DurationInt
+import BlazegraphSlowQueryStoreSuite._
+
+class BlazegraphSlowQueryStoreSuite
+    extends BioSuite
+    with IOFixedClock
+    with Doobie.Fixture
+    with Doobie.Assertions
+    with BlazegraphSlowQueryStoreFixture {
+  override def munitFixtures: Seq[AnyFixture[_]] = List(doobie, blazegraphSlowQueryStore)
+
+  private lazy val store = blazegraphSlowQueryStore()
+
+  test("Save a slow query") {
+
+    val slowQuery = BlazegraphSlowQuery(
+      view,
+      SparqlQuery(""),
+      failed = true,
+      1.second,
+      Instant.now(),
+      Identity.User("Ted Lasso", Label.unsafe("epfl"))
+    )
+
+    for {
+      _      <- store.save(slowQuery)
+      lookup <- store.listForTestingOnly(view)
+    } yield {
+      assertEquals(lookup, List(slowQuery))
+    }
+  }
+
+  test("Remove old queries") {
+    for {
+      _       <- store.save(OldQuery)
+      _       <- store.save(RecentQuery)
+      _       <- store.removeQueriesOlderThan(OneWeekAgo)
+      results <- store.listForTestingOnly(view)
+    } yield {
+      assert(results.contains(RecentQuery), "recent query was deleted")
```

Comment:
The clues are inverted

Reply:
I think it's correct, if results doesn't contain recent query, then it was deleted